packages/api/test/e2e/api.calls.e2e.ts

```diff
@@ -62,11 +63,9 @@ describe('e2e api calls', () => {
       expect(currentSession.currentEra.toNumber()).toBeGreaterThanOrEqual(0);
       expect(currentSession.currentIndex.toNumber()).toBeGreaterThanOrEqual(0);
       expect(currentSession.eraLength.toNumber()).toBeGreaterThanOrEqual(0);
-      expect(currentSession.eraProgress.toNumber()).toBeGreaterThanOrEqual(0);
```

Comment:
eraProgress and sessionProgress does not exist in api.derive.session.info anymore